### DISCUSSION TITLE
Metrics for OSSNW April - September 2022

### DISCUSSION THREAD

**Discussion by jspeed-meyers** [Open-Source-Software-Neighborhood-Watch/town-square#10] [General]
Proposal:

- Submit three pieces of writing involving at least two members of the OSSNW. Hopefully at least one is accepted.
- Submit at least two conference presentations related to the activities of the OSSNW. Hopefully at least one is accepted.

What are your thoughts? @jackiekazil, @tpike3, @MNigl, @gps-iqt-labs

**Comment by jspeed-meyers** [comment]
Friendly ping before tomorrow's meeting:  @jackiekazil, @tpike3, @MNigl

**Reply by jackiekazil** [reply]
This seems reasonable to me.

**Reply by tpike3** [reply]
I concur seems reasonable

**Comment by jspeed-meyers** [comment]
Well, the "dependency issues" piece was accepted. Yay!

**Comment by jspeed-meyers** [comment]
@tpike3, @jackiekazil mentioned this conference: http://sbp-brims.org/2022/cfp/

What about submitting the Python paper there? It seems like a potentially good fit and home.

**Reply by tpike3** [reply]
@jspeed-meyers  Which python paper?

**Reply by jspeed-meyers** [reply]
@tpike3, the one where you, Rob, and Jackie analyze the contributor networks of top Python packages, the one that you very kindly added me as a co-author.
